programs/relayer/Cargo.toml

```diff
@@ -14,7 +14,7 @@ tonic = { workspace = true, default-features = true }
 tracing            = { workspace = true, default-features = true }
 tracing-subscriber = { workspace = true, default-features = true }
 clap               = { workspace = true, default-features = true }
-anyhow             = { workspace = true, default-features = true }
+anyhow             = { workspace = true, default-features = true, features = ["backtrace"] }
```

Comment:
This enables stack trace output in the relayer logs. will be very useful for debugging in the future...